model/project_ref.go

```diff
@@ -1936,6 +1940,14 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
+func (p *ProjectRef) removeFromAdmins(user string) {
```

Comment:
We also need to handle the case that removing an admin doesn't work (i.e. we need an "addToAdminsList")

Reply:
no need for a function but added using append